### ISSUE TITLE
GWL: Strings from Hot Keys section of Settings are not in the Translation Launchpad

### ISSUE THREAD

**Issue by claudiux** [linuxmint/cinnamon#9081]
```
 * Cinnamon version 4.4.6
   - using the daily builds PPA: No
 * Distribution: Mint 19.3
 ```

**Issue**
The strings from Hot Keys section (General tab of Settings) are not in the cinnamon_cinnamon-fr.po file.

![Capture du 2020-01-01 22-47-46](https://user-images.githubusercontent.com/33965039/71646431-ed178d80-2ce8-11ea-9681-2aa5833a9a1b.png)

**Steps to reproduce**
Download at least one .po file from https://translations.launchpad.net/linuxmint/latest/+pots/cinnamon/+export

Search the strings:

1. "Hot Keys"
2. "Global hotkey for cycling through thumbnail menus"
3. "Global hotkey to show the order of apps"
4. "Duration of the apps order display on hotkey press"

**Expected behaviour**
That these strings can be translatable on Launchpad.

**Comment by icarter09** [comment]
@claudiux you still experiencing this issue? Thanks.

**Comment by claudiux** [comment]
Fixed in Cinnamon 4.6.7. Many thanks!
